### DISCUSSION TITLE
Party Sheet

### DISCUSSION THREAD

**Discussion by rwisaak** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#38] [Q&A] [answered]
Has there been any thought into adding a party tracking sheet into this? I know there's a module out there that does this already (https://github.com/maxstar/forbidden-lands-party-sheet/tree/master), but would be nice to have something fit more cleanly into the design language used here.

**Comment by aMediocreDad** [comment, marked as answer]
As long as that module exists we have no intention of duplicating the effort. I do however intend to contact the creator to bring it into the system.
